examples/getting_started/example_simulation.py

```diff
@@ -23,9 +23,20 @@
     )
 
     # Simulation the Initial Guess
+    # Interpolation: Constant
     X = InitialGuess([0, 0, 0, 0])
     U = InitialGuess([-1, 1])
 
+    sol_from_initial_guess = Solution(ocp, [X, U])
+    s = sol_from_initial_guess.integrate(shooting_type=Shooting.SINGLE_CONTINUOUS)
+    print(f"Final position of q from single shooting of initial guess = {s.states['q'][:, -1]}")
+
+    # Interpolation: Each frame
```

Comment:
I think that for explanatory case, you could add in the comment (for instance, `values from a previous optimization or from measured data`)